project/Doc.scala

```diff
@@ -76,7 +76,13 @@ object Scaladoc extends AutoPlugin {
       // Workaround https://issues.scala-lang.org/browse/SI-10028
       "-skip-packages", "org.apache.pekko.pattern:org.specs2",
       "-doc-canonical-base-url", "https://pekko.apache.org/api/pekko-http/current/") ++
-      plugins.map(plugin => "-Xplugin:" + plugin)
+      plugins.map(plugin => "-Xplugin:" + plugin) ++
+      // Workaround https://issues.scala-lang.org/browse/SI-10028
+      (if (scalaBinaryVersion == "3")
+         // https://github.com/lampepfl/dotty/issues/14939
+         List("-skip-packages:akka.pattern:org.specs2")
```

Comment:
should be 'org.apache.pekko.pattern' - this 'akka.pattern' issue appears in `else` too

Reply:
Fixed and pushed